src/AnalyzeRegistry.jl

```diff
@@ -185,7 +185,7 @@ function find_packages(; registry = general_registry())
     # Get the directories of all packages.  Filter out JLL packages: they are
     # automatically generated and we know that they don't have testing nor
     # documentation.
-    return [joinpath(registry, splitpath(p["path"])...) for (_, p) in packages if !endswith(p["name"], "_jll")]
+    return [joinpath(registry, splitpath(p["path"])...) for (_, p) in packages if !endswith(p["name"], "_jll") && p["name"] != "julia"]
```

Comment:
Perhaps check the uuid instead of the name (or both)? Even though I guess it's unlikely another registry will have a package called "Julia". Also, please add a comment above explaining why we drop Julia (it isn't a real package, etc...)

Reply:
do you know why it exists at all?

Reply:
If I remember correctly, they wanted it as a way to "upgrade Julia via the package manager", functionality never implemented and unlikely to happen anytime soon. This entry has been also removed once or twice already, but that wrecked havoc (you can't remove packages from the registry), so we have to keep it

Reply:
ah ok, makes sense